services/order.py

```diff
@@ -0,0 +1,34 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from django.utils.dateparse import parse_datetime
+from django.utils.timezone import now
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+@transaction.atomic()
+def create_order(
+    tickets: list[dict],
+    username: str = None,
+    date: str = None
+) -> None:
+    created_at = parse_datetime(date) if date else now()
+    order = Order.objects.create(user=User.objects.get(username=username))
```

Comment:
Consider adding error handling for the case where the `User.objects.get(username=username)` does not find a user, which would raise a `DoesNotExist` exception.